.rubocop.yml

```diff
@@ -30,17 +30,9 @@ GetText/DecorateString:
   - spec/**/*
 RSpec/BeforeAfterAll:
   Description: Beware of using after(:all) as it may cause state to leak between tests.
-    A necessary evil in acceptance testing. Disabled here because there is setup that
-    needs to be done before all of the examples in the context but it would be wasteful
-    to do that set up for each example in the context.
+    A necessary evil in acceptance testing.
```

Comment:
Btw, this isn't my wording here. This is the default wording and I'm just restoring it.